assets/src/scripts/components/hydrograph/graph-controls.spec.js

```diff
@@ -0,0 +1,243 @@
+import {Actions, configureStore} from '../../store';
+import {attachToNode} from './index';
```

Comment:
See below. You don't need this import. I'm a little surprised that the test works with this imported.

Reply:
done!